create_database.py

```diff
@@ -0,0 +1,42 @@
+import mysql.connector
+from mysql_details import *
+
+
+def create_db():
+    try:
+        # create connection object
+        conn=mysql.connector.connect(host=Host,user=Username,passwd=Password)
+
+        #creating cursor object
+        mycursor=conn.cursor()
+
+        #create Database
+        mycursor.execute("CREATE DATABASE bikepoints")
+
+        #use database bikepoints
+        mycursor.execute("USE bikepoints")
+
+        #create table "bikepoints data"  for storing real time bike availability data across London
+        mycursor.execute("""CREATE TABLE `bikepoints data` (

```

Comment:
why not use SQLAlchemy to create your table as well?

Reply:
Actually, initially I started working with "mysql.connector" and later I realized that I need SQLALchemy engine object for storing table using pandas. I'll make changes and use SQLALchemy.